extensions/default/src/DicomWebDataSource/utils/StaticWadoClient.ts

```diff
@@ -211,6 +231,9 @@ export default class StaticWadoClient extends api.DICOMwebClient {
    * @returns
    */
   filterItem(key: string, queryParams, study, sourceFilterMap) {
+    const { supportsFuzzyMatching = false } = this.config;
+    const isPatientName = key === 'patientname';
```

Comment:
Could you make it key.indexof('name')!==-1 so that it works for other names as well?

Reply:
done